pkg/types/baremetal/platform.go

```diff
@@ -19,6 +19,7 @@ type Host struct {
 	Role            string `json:"role"`
 	BootMACAddress  string `json:"bootMACAddress" validate:"required,uniqueField"`
 	HardwareProfile string `json:"hardwareProfile"`
+	RootHint        string `json:"rootHint"`
```

Comment:
is this required? or optional?if optional does it have a default? any restrictions on the value?

any other documentation that someone looking at the godoc can read and understand a bit more.

Reply:
thanks, I'll update the patch once we figure out if this is something we want to support. Will put a hold on it in the meantime.